src/sdk/Exports.ts

```diff
@@ -69,3 +69,4 @@ export { SpeechSynthesizer } from "./SpeechSynthesizer";
 export { SpeechSynthesisResult } from "./SpeechSynthesisResult";
 export { SpeechSynthesisEventArgs } from "./SpeechSynthesisEventArgs";
 export { SpeechSynthesisWordBoundaryEventArgs} from "./SpeechSynthesisWordBoundaryEventArgs";
+export { SpeakerAudioDestination } from "../common.browser/SpeakerAudioDestination";
```

Comment:
Objects that the SDK is going to expose should be in the SDK folder.

All the other folders are internal details.

Reply:
Thanks, move the `SpeakerAudioDestination` to `sdk/Audio`